cmd/eval.go

```diff
@@ -290,6 +291,7 @@ access.
 	evalCommand.Flags().BoolVar(&params.disableIndexing, "disable-indexing", false, "disable indexing optimizations")
 	evalCommand.Flags().BoolVar(&params.disableEarlyExit, "disable-early-exit", false, "disable 'early exit' optimizations")
 	evalCommand.Flags().BoolVarP(&params.strictBuiltinErrors, "strict-builtin-errors", "", false, "treat built-in function errors as fatal")
+	evalCommand.Flags().BoolVarP(&params.showBuiltinErrors, "show-builtin-errors", "", false, "show all built-in function errors in output")
```

Comment:
[subjective/opinion] I'd suggest changing the help text to give the user a little more context about what's going on when they flip on the CLI option? I'm hoping we can communicate more about what distinguishes `show-builtin-errors` from `strict-builtin-errors`.

Maybe changing "show all built-in errors" -> "collect and display all built-in errors"?

Again, this is a somewhat subjective comment, so it's entirely optional to address it if you think the original wording works better. :slightly_smiling_face:

Reply:
I've tried to make this clearer in d34d4223 by talking about 'encountered errors'. Let me know if that makes more sense to you too.

Reply:
Oooh! :smiley: I really like that choice of wording. Thank you for tweaking that message text!